### PR TITLE
add resource links to readme

### PR DESCRIPTION
**Pull request by dannyvankooten**
This pulls from a few other popular cryptocurrency README's by adding a **Resources** section with links to the various available resources, like the Discord chat or the RaiBlocks forum. 

I had several people ask me for instructions on how to run a node so I've added in several links to the wiki, since that's where the bulk of this information can be found right now.

### CODE REVIEW

**Comment by donutloop** [comment]
We do have a little content conflict 

https://github.com/clemahieu/raiblocks/pull/431/files

File: README.md [hunk 1]
```diff
@@ -1,29 +1,32 @@
-[![Build Status](https://travis-ci.org/clemahieu/raiblocks.svg?branch=master)](https://travis-ci.org/clemahieu/raiblocks) 
-___
-
+RaiBlocks
```
Comment (dhimmel) [inline README.md]: Use `# RaiBlocks` here rather than `====` for consistency?

Consider also blank newline after all headers.

File: README.md [hunk 2]
```diff
@@ -1,29 +1,32 @@
...
 * Offers feeless, instantaneous transactions, as well as unlimited scalability, making RaiBlocks ideal for peer-to-peer transactions.
 * As of December 2017, the RaiBlocks network has processed over four million transactions with an unpruned ledger size of only 1.7GB.
 
-***
+For more information, see [RaiBlocks.net](https://raiblocks.net/) or read the [RaiBlocks whitepaper](https://raiblocks.net/media/RaiBlocks_Whitepaper__English.pdf). 
 
-### For more information:
-- [Homepage](https://raiblocks.net)
+### Resources
+- [RaiBlocks website](https://raiblocks.net)
```
Comment (dhimmel) [inline README.md]: Nice!

File: README.md [hunk 3]
```diff
@@ -1,29 +1,32 @@
...
+- [Discord chat](https://chat.raiblocks.net)
+- [RaiBlocks forum](https://forum.raiblocks.net/)
+- [GitHub wiki](https://github.com/clemahieu/raiblocks/wiki)
 
 ### Build instructions: 
 https://github.com/clemahieu/raiblocks/wiki/Build-Instructions
+
+### Need help?
+- See the documentation at the [RaiBlocks wiki](https://github.com/clemahieu/raiblocks/wiki) for help and more information.
+- Ask for help in the [Discord #development](https://discordapp.com/channels/370266023905198083/370285507185344524) channel.
```
Comment (androm3da) [inline README.md]: Please don't put the dev channel here.

File: README.md [hunk 4]
```diff
@@ -1,29 +1,32 @@
...
-
-***
+- [Discord chat](https://chat.raiblocks.net)
+- [RaiBlocks forum](https://forum.raiblocks.net/)
+- [GitHub wiki](https://github.com/clemahieu/raiblocks/wiki)
 
 ### Build instructions: 
 https://github.com/clemahieu/raiblocks/wiki/Build-Instructions
+
+### Need help?
```
Comment (androm3da) [inline README.md]: Qualify this as "development help".

e.g. "Need help getting started on developing with raiblocks?"

File: README.md (hunk 3 above)
Comment (androm3da) [inline README.md]: Add the contributor's guide from the wiki

Comment (dannyvankooten) [reply README.md]: Done 👍

Comment (dannyvankooten) [reply README.md]: Gotcha

File: README.md (hunk 4 above)
Comment (dannyvankooten) [reply README.md]: Got rid of this section altogether as the #431 already links to Discord and the wiki is linked under "Resources", so it seemed a bit repetitive.

**Comment by dannyvankooten** [comment]
Is there some way I can mark the requested changes as handled? I'm not sure... 🤔

**Comment by dhimmel** [comment]
> Is there some way I can mark the requested changes as handled? I'm not sure...

It's up to reviewer who requested changes to re-review the PR with the new commits and approve if the issues have been addressed. If this doesn't happen, you can always mention the user (like @androm3da), so they're more likely to get notified.

**Pull request merged**